src/app/(BottomTabNavigation)/Profile/index.tsx

```diff
@@ -1,36 +1,127 @@
 import { router } from 'expo-router';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Text, TouchableOpacity, View } from 'react-native';
 
 import styles from './styles';
-import { signOutUser } from '../../../supabase/queries/auth';
+import Envelope from '../../../../assets/envelope.svg';
+import GreyRightCarrot from '../../../../assets/grey-right-carrot.svg';
+import Location from '../../../../assets/location.svg';
+import Pencil from '../../../../assets/pencil.svg';
+import Person from '../../../../assets/person.svg';
+import RedTrash from '../../../../assets/red-trash.svg';
+import Reset from '../../../../assets/reset.svg';
+import SignOut from '../../../../assets/sign-out.svg';
+import CasesHeader from '../../../Components/CasesHeader/CasesHeader';
+import {
+  getCurrentUserInfo,
+  signOutUser,
+} from '../../../supabase/queries/auth';
+import { User, userInstance } from '../../../types/types';
 
 function ProfileScreen() {
+  const [currSession, setCurrSession] = useState<User>(userInstance);
```

Comment:
When we navigate back to this screen after updating a user's information, the fields are not updated. I think we can address this when we implement Auth Context though.

Reply:
yes I think currSession stuff will be addressed when I do auth context 🏄🏾‍♀️